arbitrum-docs/node-running/running-a-node.mdx

```diff
@@ -104,19 +104,71 @@ Note: If you’re interested in accessing an Arbitrum chain, but you don’t wan
 - The feed relay is in the same docker image
 - Here is an example of how to run the feed relay for Arbitrum One:
   ```shell
-  docker run --rm -it  -p 0.0.0.0:9642:9642 --entrypoint relay offchainlabs/nitro-node:v2.0.10-73224e3 --node.feed.output.addr=0.0.0.0 --node.feed.input.url=wss://arb1.arbitrum.io/feed --l2.chain-id=42161
+  docker run --rm -it  -p 0.0.0.0:9642:9642 --entrypoint relay offchainlabs/nitro-node:v2.0.11-8e786ec --node.feed.output.addr=0.0.0.0 --node.feed.input.url=wss://arb1.arbitrum.io/feedy --l2.chain-id=42161
   ```
 - Here is an example of how to run nitro-node for Arbitrum One with custom relay:
   ```shell
-  docker run --rm -it  -v /some/local/dir/arbitrum:/home/user/.arbitrum -p 0.0.0.0:8547:8547 -p 0.0.0.0:8548:8548 offchainlabs/nitro-node:v2.0.10-73224e3 --l1.url=https://l1-mainnet-node:8545 --l2.chain-id=42161 --http.api=net,web3,eth,debug --http.corsdomain=* --http.addr=0.0.0.0 --http.vhosts=* --node.feed.input.url=ws://local-relay-address:9642
+  docker run --rm -it  -v /some/local/dir/arbitrum:/home/user/.arbitrum -p 0.0.0.0:8547:8547 -p 0.0.0.0:8548:8548 offchainlabs/nitro-node:v2.0.11-8e786ec --l1.url=https://l1-mainnet-node:8545 --l2.chain-id=42161 --http.api=net,web3,eth,debug --http.corsdomain=* --http.addr=0.0.0.0 --http.vhosts=* --node.feed.input.url=ws://local-relay-address:9642
   ```
 
-### Running a Validator
+### Types of Validators
 
 - Currently, the ability to post assertions on-chain for mainnet Arbitrum chains is whitelisted. However, anyone can run a validator in `Watchtower` mode which will immediately log an error if an on-chain assertion deviates from the locally computed chain state
-- Running a validator in `Watchtower` mode is the same as running an archive Nitro node with `--node.validator.enable --node.validator.strategy=Watchtower`
-- Here is an example of how to run validator for Arbitrum One:
+- Full list of validator strategies:
+  * `Watchtower`
+    - If local state disagrees with on-chain assertion, simply log an error (no wallet required, gas never spent on-chain)
+  * `Defensive` (Whitelist required)
+    - Post stake and create challenge if local state disagrees with on-chain assertion (wallet required, will only post stake on-chain if bad assertion found)
+  * `StakeLatest` (Whitelist required)
+    - Stay staked on latest node and challenge any bad assertions found (wallet required, always staked, uses some gas every time new assertion created)
+  * `ResolveNodes` (Whitelist required)
+    - Stay staked on latest node, resolve any unconformed nodes and challenge any bad assertions found (wallet required, always staked, uses some gas every time unconfirmed node resolved or new assertion created)
```

Comment:
To avoid overloading the term term "nodes," how about we strictly call them "assertions" here?

Reply:
:+1: